packages/babel-parser/src/parser/statement.js

```diff
@@ -2041,13 +2041,31 @@ export default class StatementParser extends ExpressionParser {
     // import '...'
     node.specifiers = [];
     if (!this.match(tt.string)) {
+      // check if we have a default import like
+      // import React from "react";
       const hasDefault = this.maybeParseDefaultImportSpecifier(node);
+      /* we are checking if we do not have a default import, then it is obvious that we need named imports
+       * import { get } from "axios";
+       * but if we do have a default import
+       * we need to check if we have a comma after that and
+       * that is where this `|| this.eat` condition comes into play
+       */
```

Comment:
@vivek12345 I don't know if I already thanked you, but I love these comments!

Reply:
Thank you @nicolo-ribaudo  . Means a lot 🙏